WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -479,50 +470,52 @@ import Foundation
             let button = buttons.removeLast() as UIButton
             configureCommentActionButton(button)
         }
-
-        // Show visit
-        if UIDevice.isPad() {
-            let button = buttons.removeLast() as UIButton
-            configureVisitActionButton(button)
-        } else {
-            configureVisitActionButton(actionButtonFlushLeft)
-        }
     }
 
     private func resetActionButtons() {
-        resetActionButton(actionButtonCenter)
-        resetActionButton(actionButtonFlushLeft)
         resetActionButton(actionButtonLeft)
         resetActionButton(actionButtonRight)
     }
 
     private func resetActionButton(button:UIButton) {
         button.setTitle(nil, forState: .Normal)
+        button.setTitle(nil, forState: .Highlighted)
         button.setImage(nil, forState: .Normal)
         button.setImage(nil, forState: .Highlighted)
         button.selected = false
         button.hidden = true
     }
 
-    private func configureActionButton(button: UIButton, title: String?, image: UIImage?, highlightedImage: UIImage?) {
+    private func configureActionButton(button: UIButton, title: String?, image: UIImage?, highlightedImage: UIImage?, selected:Bool) {
         button.setTitle(title, forState: .Normal)
+        button.setTitle(title, forState: .Highlighted)
         button.setImage(image, forState: .Normal)
         button.setImage(highlightedImage, forState: .Highlighted)
-        button.selected = false
+        button.selected = selected
         button.hidden = false
     }
 
     private func configureLikeActionButton(button: UIButton) {
         button.tag = CardAction.Like.rawValue
         let likeStr = NSLocalizedString("Like", comment: "Text for the 'like' button. Tapping marks a post in the reader as 'liked'.")
-        let likedStr = NSLocalizedString("Liked", comment: "Text for the 'like' button. Tapping removes the 'liked' status from a post.")
-        let title = contentProvider!.isLiked() ? likedStr : likeStr
+        let likesStr = NSLocalizedString("Likes", comment: "Text for the 'like' button. Tapping removes the 'liked' status from a post.")
+
+        let count = contentProvider!.likeCount().integerValue
+
+        var title: String?
```

Comment:
lol, no worries... it's actually a very minor thing and I only pointed it out because I knew it'd take me a while to test this right.  Thanks for the patience :+1: